File: JavaScript/ReverseArray.js
function ReverseArray(arr){
    let left=0
    let right=arr.length-1
    while(right> left){
        let temp=arr[left]
        arr[left]=arr[right]
        arr[right]=temp
        left++
        right--
    }
    return arr
}
console.log(ReverseArray([2,3,4,5,6,8]))

File: python/ReverseArray.py
# def ReverseArray(arr):
#     return arr[::-1]
# print(ReverseArray([3,4,5,6,7,8]))

def ReverseArray(arr):
    left=0
    right=len(arr)-1
    for i in range(len(arr)):
        if right > left:
            arr[left], arr[right]=arr[right], arr[left]
            left += left+1
            right -= right-1
    return arr
print(ReverseArray([2,3,4,5,6]))